<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<node name="tb_abmap_frontier_node" pkg="tb_abmap" type="tb_abmap_frontier_node" respawn="true">
		<param name="resolution" type="double" value="1.0" />
	</node>
	<node name="tb_abstractmap_node" pkg="tb_abmap" type="tb_abstractmap_node" respawn="true" output="screen">
	<param name="write_heightimage_interval" type="double" value="3.0" />
	<param name="grid_sidelength" type="double" value="10.0" />
	</node>

</launch>
